### PR TITLE
Logging of packet size in OutOfProc communication

### PR DESCRIPTION
**Pull request by ostorc**
This PR adds logging of packet size in node communication.

### CODE REVIEW

File: src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@
 using System.Threading.Tasks;
 using Microsoft.Build.Shared.FileSystem;
 using Microsoft.Build.Utilities;
+using Microsoft.Build.Eventing;
```
Comment (rainersigwald) [inline src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs:22]:
```suggestion
using Microsoft.Build.Eventing;
using Microsoft.Build.Shared.FileSystem;
using Microsoft.Build.Utilities;
```

Comment (ostorc) [reply src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs:22]: What about this? Merging the 3 MSBuild using statements with previous block of MSBuild using statements and removing unused Threading statement.
 ```
-using Microsoft.Build.Shared;
+using Microsoft.Build.Eventing;
 using Microsoft.Build.Exceptions;
 using Microsoft.Build.Internal;
-
-using BackendNativeMethods = Microsoft.Build.BackEnd.NativeMethods;
-using System.Threading.Tasks;
+using Microsoft.Build.Shared;
 using Microsoft.Build.Shared.FileSystem;
 using Microsoft.Build.Utilities;
-using Microsoft.Build.Eventing;
+
+using BackendNativeMethods = Microsoft.Build.BackEnd.NativeMethods;
```
(don't know how to properly format it, this is output from git diff)

Comment (rainersigwald) [reply src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs:22]: Yes, that looks great.

For formatting, there are two options:

This button:
image

or starting a code block with the language `diff`

```markdown
```diff
+whatever
-something
```

Comment (ostorc) [reply src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs]: I tried that button, but the result felt confusing. I didn't know about the `diff`, thanks.

**Pull request merged**